admin-dev/themes/default/template/controllers/orders/helpers/view/view.tpl

```diff
@@ -709,11 +709,19 @@
                 <div class="well">
                   <div class="row">
                     <div class="col-sm-6">
+                      <a class="btn btn-default pull-right" style="margin-left: 5px" onclick="copyToClipboard('{displayAddressDetail address=$addresses.delivery newLine='\n'}', event);">
+                        <i class="icon-copy"></i>
+                        {l s='Copy'}
+                      </a>
                       <a class="btn btn-default pull-right" href="?tab=AdminAddresses&amp;id_address={$addresses.delivery->id}&amp;addaddress&amp;realedit=1&amp;id_order={$order->id}&amp;address_type=1&amp;token={getAdminToken tab='AdminAddresses'}&amp;back={$smarty.server.REQUEST_URI|urlencode}">
                         <i class="icon-pencil"></i>
                         {l s='Edit'}
                       </a>
-                      {displayAddressDetail address=$addresses.delivery newLine='<br />'}
+
+                      <div ondblclick="copyDivTextContentToClipboard(event, true);" data-placement="left" data-content="{l s='Copied!'}" data-trigger="manual">
+                      {displayAddressDetail address=$addresses.delivery newLine='</div><div ondblclick="copyDivTextContentToClipboard(event, true);" data-placement="left" data-content="Copied!" data-trigger="manual">'}
```

Comment:
How does one use the `l` function inside of something like this? Should definitely pull variations of `Copied!` from the language. I'm just not sure how.